Sources/SwiftMemcache/Extensions/UInt8+Characters.swift

```diff
@@ -11,3 +11,10 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 //===----------------------------------------------------------------------===//
+
+
+extension UInt8 {
+    static let whitespace: UInt8 = UInt8(ascii: " ")
+    static let newline = UInt8(ascii: "\n")
+    static let carriageReturn = UInt8(ascii: "\r")
```

Comment:
Let's make these static vars instead. The compiler is often better optimising those